README.md

```diff
@@ -195,14 +195,14 @@ By default this type of reporting is not enabled.
 
 ##### include_logs_catalog_failure
 
-`Boolean`: Include logs if a catalog fails to compile. This is a more specific type of failure that indicates a server-side issue.
+`Boolean`: Include logs if a [catalog](https://puppet.com/docs/puppet/7/man/catalog.html) fails to compile. This is a more specific type of failure that indicates a server-side issue.
```

Comment:
This link is less about the catalog itself and more about the catalog sub command. If you can find a link describing what a catalog really is then feel free to put it here, otherwise I would just drop this one.